### ISSUE TITLE
Welcome any related paper.

### ISSUE THREAD

**Issue by XiaoYee** [XiaoYee/Awesome_Efficient_LRM_Reasoning#2]
In this issue, we welcome any paper that relates to efficient reasoning in LLM or LRM. 

Hope for your contribution.

**Comment by XuandongZhao** [comment]
Thanks for summarizing the latest work! I’d like to share our paper:
Scalable Best-of-N Selection for Large Language Models via Self-Certainty
Paper: https://arxiv.org/abs/2502.18581
Code: https://github.com/backprop07/Self-Certainty

**Comment by XiaoYee** [comment]
Thanks for your great work. We have added it.

**Comment by yubinCloud** [comment]
I’d like to share our paper: **Long-Short Chain-of-Thought Mixture Supervised Fine-Tuning Eliciting Efficient Reasoning in Large Language Models**

- paper: https://arxiv.org/abs/2505.03469

I think this work should be classified under "Reasoning Chain Compression" which falls under "Efficient Reasoning with SFT".

**Comment by XiaoYee** [comment]
@yubinCloud We have added this article to our GitHub repository and will describe your paper in the next version of the survey.

**Comment by woominsong** [comment]
Hi, thanks for the great work! I wanted to share the following paper on efficient reasoning.

Title: Accelerated Test-Time Scaling with Model-Free Speculative Sampling
arxiv: https://arxiv.org/abs/2506.04708

This paper introduces a model-free speculative decoding method to accelerate test-time scaling without affecting its outputs, by utilizing the N-gram overlaps across different reasoning trajectories. I believe this will be a useful addition to the list!

I have also created a pull request #4 for easier integration.

**Comment by YuxuanJiang1** [comment]
Hi, thank you for the great survey! We’d like to suggest our recent work as a relevant reference:
Title: DRP: Distilled Reasoning Pruning with Skill-aware Step Decomposition for Efficient Large Reasoning Models
Link: [[arXiv link ](https://arxiv.org/abs/2505.13975)]
Summary: DRP employs a teacher model to perform skill-aware step decomposition and content pruning, and distills the pruned reasoning paths into a student model for efficient reasoning.

We believe this work may complement the topics discussed in your survey. Thank you for considering it!

**Comment by MozerWang** [comment]
Thanks for this amazing work! Here, I would like to recommend paper on Adaptive Social Reasoning of Language Agents, hoping it meet your requirements!

We propose ASL, the first adaptive social learning framework for language agents, which consists of hierarchical reasoning modes and tailor-designed reinforcement learning to empower language agents with adaptive reasoning ability in a rich social context.  We develop the AMPO algorithm, which integrates mode-level and sample-level advantage estimation for dynamic mode switching, and improves flexible inference and token efficiency.

Paper: Adaptive Thinking via Mode Policy Optimization for Social Language Agents https://arxiv.org/pdf/2505.02156
Code: https://github.com/MozerWang/AMPO

Thanks again!
